crates/examples/microkit/http-server/pds/virtio-net-driver/src/main.rs

```diff
@@ -73,114 +71,21 @@ fn init() -> HandlerImpl {
         )
     };
 
-    dev.ack_interrupt();
+    dev.irq_ack();
     DEVICE.irq_ack().unwrap();
 
-    HandlerImpl {
+    PhyDeviceHandler::new(
         dev,
         client_region,
         client_client_dma_region_paddr,
         rx_ring_buffers,
         tx_ring_buffers,
-    }
+        CLIENT,
+        DEVICE,
```

Comment:
The order of these two arguments is different from their order in `PhyDeviceHandler::new`.

Reply:
Nice catch! Yeah, that seems to have fixed it.